src/awsexplorer/developerTools.ts

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
I think in code we should consider naming this `src/awsexplorer/localExplorer.ts` or something like that. "Developer Tools" has potential for perpetual ambiguity with our other "Toolkit dev-mode" stuff.

Reply:
Yeah couldn't think of a good name. Local is a good adjective to use internally though.